src/main/java/com/github/manolo8/darkbot/config/actions/values/StatTypeValue.java

```diff
@@ -35,11 +39,18 @@ public class StatTypeValue implements Value<Number>, Parser {
         }
     }
 
-    private Stats.General getKeyFromString(String key) {
-        for (Stats.General stat : Stats.General.values()) {
-            if (stat.name().equalsIgnoreCase(key)) return stat;
-        }
-        return null;
+    private StatsAPI.Key getKeyFromString(String token) {
+        String[] tokenParts = token.split(":");
+
+        String statNamespace = tokenParts.length == 3 ? tokenParts[0] : null;
+        String statCategory = tokenParts.length >= 2 ? tokenParts[tokenParts.length - 2] : null;
+        String statKey = tokenParts[tokenParts.length - 1];
+
+        return StatsManager.getStatKeys().stream()
+                .filter(s -> statNamespace == null || s.namespace().equals(statNamespace))
```

Comment:
a null namespace is just, the null namespace. that's the bot's one, no need to resolve it